.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Ask a question
+about: Please use the forum (https://groups.google.com/forum/#!forum/cesium-dev) for general questions about using Cesium. 
+---
+
+:exclamation: Please use the [forum](https://groups.google.com/forum/#!forum/cesium-dev) for asking questions about how to use Cesium and best practices. :exclamation:
```

Comment:
Let's folks know that the core Cesium team monitors the forum and answers questions - to give them motivation and assurance to use it..